editor/scripts/bundle.py

```diff
@@ -356,6 +356,14 @@ def remove_platform_files_from_archive(platform, jar):
                 continue
             # anything else should be removed
             files_to_remove.append(file)
+        # find files to remove `.pdb` from _updack/*
+        if file.startswith("_unpack"):
+            # we need these files for windows
+            if platform == "x86_64-win32":
+                continue
+            if file.endswith(".pdb"):
+                files_to_remove.append(file)
```

Comment:
We can Build&Run `exe` file only on Windows. That means we don't need `pdb` files in the other editor bundles

Reply:
When I think about it a bit more, I understand that we cleanup `_unpack` folder and keep executables and libs only for the platform we bundle